server/mock/datastore_mock.go

```diff
@@ -674,6 +674,8 @@ type GetHostScriptExecutionResultFunc func(ctx context.Context, execID string) (
 
 type ListPendingHostScriptExecutionsFunc func(ctx context.Context, hostID uint, ignoreOlder time.Duration) ([]*fleet.HostScriptResult, error)
 
+type GetScriptResultFunc func(ctx context.Context, scriptID uint) (*fleet.ScriptResult, error)
```

Comment:
I don't think this change is required? I.e. no method was added to the `Datastore` interface, this might've been generated at a time when a datastore method was added and then it was removed but not re-generated?

Reply:
thanks!!